mbq/ranch/stats/rabbitmq.py

```diff
@@ -17,18 +17,29 @@ def send_rabbitmq_queue_stats(broker_url, queue_names):
     username = parsed.username or "guest"
     password = parsed.password or "guest"
 
+    error_type = None
     try:
         response = requests.get(url, auth=(username, password), timeout=0.1)
-    except requests.exceptions.RequestException:
+    except requests.exceptions.Timeout:
         # We set a very aggressive timeout on the request so that we do not
         # block our worker for very long. It's expected that this will
         # sometimes fail, but it's unlikely to matter since we should be
         # collecting queue stats frequently. As such, we log but otherwise
         # swallow the exception (so we do not Rollbar or emit generic error
         # metrics)
-        logger.info("Error requesting metrics from RabbitMQ, skipping")
```

Comment:
In some services (invoicing-dev) the logs are spewing and it's not helpful. Breaking out the error type and tracking on the metric. We can add logging if we need to do additional investigation once we know what errors are being thrown.